src/util/export.ts

```diff
@@ -37,6 +37,9 @@ export interface ExportOptions {
   height?: number;
   handleSuccess?: () => void;
   handleError?: (error: unknown) => void;
+  pixelratio?: number;
+  controlDiv?: HTMLElement | null;
+  zoomDiv?: HTMLElement | null;
```

Comment:
Not appropriate here. Add an array parameter instead providing the elements that may be excluded from the export.